include/glow/Partitioner/Partitioner.h

```diff
@@ -96,26 +99,20 @@ class Partitioner {
                                        llvm::StringRef backendName,
                                        NodeToFunctionMap &mapping);
 
-  /// Given the node-function mapping, do the actual partitioning. If \p saveDAG
-  /// is true, the DAG will be saved into partitions_, which is the final
-  /// partition result.
-  void doPartitioning(llvm::StringRef funcName, std::vector<Function *>,
-                      NodeToFunctionMap &mapping, bool saveDAG);
-
   /// If there is no need to do any partition, just generate the DAGNode based
   /// on current functions in this module for backend \p backendName found in \p
   /// backendMap. \p cctx is used during optimization of the Function. \returns
   /// whether there was an error encountered.
```

Comment:
nit: This doesn't return if an error was encountered, it returns the DAG list or else an error.